presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/credential/ExtraCredentialConfig.java

```diff
@@ -11,31 +11,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.prestosql.plugin.jdbc;
+package io.prestosql.plugin.jdbc.credential;
 
 import io.airlift.configuration.Config;
-import io.prestosql.plugin.jdbc.credential.CredentialProviderType;
-
-import javax.validation.constraints.NotNull;
 
 import java.util.Optional;
 
-import static io.prestosql.plugin.jdbc.credential.CredentialProviderType.INLINE;
-import static java.util.Objects.requireNonNull;
-
-public class BaseJdbcAuthenticationConfig
+public class ExtraCredentialConfig
```

Comment:
ExtraCredentialAuthenticationConfig or ExtracCredentialCredentialConfig? Because we are using extra credentials infra to pass to authentication credentials.

Reply:
Either of those seem redundant to me. This class configures how we use "extra credentials" for the JDBC connector. The phrase "authentication credential" seems redundant given that the purpose of a credential is to be used for authentication.

I'll leave this for now, but we can follow up and rename if you feel strongly about it.